src/sandstorm/update-tool.capnp

```diff
@@ -0,0 +1,54 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0x96c3fff3f4beb8fe;
+# This file contains schemas relevant to the Sandstorm self-updater.
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+struct PublicSigningKey {
+  key0 @0 :UInt64;
```

Comment:
I think this would ideally be a fixed-size list of type `UInt8[32]`, but unfortunately those aren't implemented yet for Cap'n Proto.